src/functTest/groovy/com/bmuschko/gradle/docker/DockerRemoteApiPluginFunctionalTest.groovy

```diff
@@ -261,6 +261,38 @@ class DockerRemoteApiPluginFunctionalTest extends AbstractGroovyDslFunctionalTes
         build('convert')
     }
 
+
+    def "configuration cache compatible when docker state changes on disk for OS #osName"() {
+        given:
+        useGradleVersion("8.3")
+
+        and:
+        // Force use of user.home and define a variable holding the file
+        def home = new File(temporaryFolder, "home").tap{it.mkdirs()}
+        def dockerSockDirectory = new File(home, "/.docker/run/").tap{it.mkdirs()}
+        String[] arguments = [
+            "-Dcom.bmuschko.gradle.docker.internal.DefaultDockerUrlValueSource.skipCheckOfVarRun=true",
```

Comment:
⭕ please extract a const - it's used in at least three places

Reply:
the thing is, I do not want to expose it. It is used in 2 places, one in the test and one in the class